### DISCUSSION TITLE
Article: Review of Parcel version 2: Now on front page of hacker news!!

### DISCUSSION THREAD

**Discussion by cekvenich2** [parcel-bundler/parcel#6826] [General]
https://medium.com/@vic850c/preview-of-parcel-version-2-eb986f6d4d67

**Comment by cekvenich2** [comment]
The designers of v2 disagree with the post, without giving a reason:
- https://github.com/parcel-bundler/parcel/issues/6822

**Reply by DeMoorJasper** [reply]
Because it wasn't an issue, just shameless self-promotion... it's even behind the medium pay-wall...

As Devon already mentioned you should create a separate issue for each problem you encounter.

It is fine in this general discussion though, but issues is not a place for promoting articles, that's for bugs and feature requests

There's also a bunch of complaining in that article for which most is just due to not really knowing how parcel 2 works
